db/migrate/20150313054707_create_translations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTranslations < ActiveRecord::Migration
+  def change
+    create_table :translations do |t|
+      t.datetime :time_assigned
+
```

Comment:
will look into it and fix.

On Mon, Mar 16, 2015 at 12:58 AM, Darren Zhao <notifications@github.com>
wrote:

> In db/migrate/20150313054707_create_translations.rb
> <https://github.com/DarryQueen/Khan-Burmese/pull/40#discussion_r26465646>:
>
> > @@ -0,0 +1,9 @@
> > +class CreateTranslations < ActiveRecord::Migration
> > +  def change
> > +    create_table :translations do |t|
> > +      t.datetime :time_assigned
> > +
>
> No, those are fine and belong in the model, not the schema. The issue is
> that you don't have a t.references field, so your translations have no
> way to identify the corresponding video and user in the database.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarryQueen/Khan-Burmese/pull/40/files#r26465646>.
>